ProjectSettings/ProjectSettings.asset

```diff
@@ -855,7 +855,7 @@ PlayerSettings:
     m_VersionCode: 1
     m_VersionName: 
   apiCompatibilityLevel: 6
-  activeInputHandler: 1
+  activeInputHandler: 2
```

Comment:
@nathanscommits can you revert this project setting from the menu?

Reply:
Oh yeah forgot about that, will do!